modules/lily-http/src/main/java/io/github/tomboyo/lily/http/encoding/Encoding.java

```diff
@@ -15,25 +18,82 @@ public class Encoding {
           .configure(WRITE_DATES_AS_TIMESTAMPS, false);
 
   private static final ObjectMapper formExplodeMapper =
-      new ObjectMapper(new FormExplodeFactory())
+      new ObjectMapper(new FormExplodeFactory("?"))
+          .registerModule(new JavaTimeModule())
+          .configure(WRITE_DATES_AS_TIMESTAMPS, false);
+
+  private static final ObjectMapper formContinuationExplodeMapper =
+      new ObjectMapper(new FormExplodeFactory("&"))
           .registerModule(new JavaTimeModule())
           .configure(WRITE_DATES_AS_TIMESTAMPS, false);
 
   private Encoding() {}
 
-  public static String simple(Object o) {
-    try {
-      return simpleMapper.writer().writeValueAsString(o);
-    } catch (JsonProcessingException e) {
-      throw new UncheckedIOException(e);
+  public enum Modifiers {
+    EXPLODE;
+  }
+
+  /**
+   * Returns an encoder which implements RFC6570 simple-style string expansion, for the {@code
```

Comment:
Let's remove the `{var}` from documentation as it's probably more confusing than helpful.